besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1974,12 +1996,28 @@ private void checkPrivacyTlsOptionsDependencies() {
           "Known-clients file must be specified or CA clients must be enabled when TLS client authentication is enabled for JSON-RPC HTTP endpoint");
     }
 
+    rpcHttpTlsProtocols.retainAll(getJDKEnabledProtocols());
+    if (rpcHttpTlsProtocols.isEmpty()) {
+      throw new ParameterException(commandLine, "No valid TLS protocols specified");
+    }
+
+    for (String ciphersuite : rpcHttpTlsCipherSuites) {
```

Comment:
why the validations of the cipherSuites is implemented differently from the tlsProtocol?

Reply:
There are only few protocols available (for instance on my jdk ["TLSv1.3", "TLSv1.2", "SSLv2Hello"]), their names are short and clear and you would only only diverge from default in legacy systems I would think. It makes sense to me to just show 1 error telling you that you did not specify any supported protocol I will update the error to show user which protocols are supported.


On the other hand when specifying the suite it requires precision so it makes sense to tell the user about each mistake explicitly. There are 48 enabled suites on my jdk. 

If you think I should remake it some other way, let me know 👍

Reply:
thanks for clarify, now it make sense the different approach